packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -427,11 +515,108 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
     const { contentContainer } = this;
 
     if (contentContainer) {
-      this._stickies.forEach((sticky: Sticky) => {
-        sticky.syncScroll(contentContainer);
+      // sync Sticky scroll if contentContainer has scrolled horizontally
+      if (this._scrollLeft !== contentContainer.scrollLeft) {
```

Comment:
These references to `scrollLeft` would cause reflow. They should be wrapped in `safeRequestAnimationFrame` calls

Reply:
I think, reading scrollLeft/scrollTop in 'scroll' event handler for contentContainer won't cause reflow because contentContainer would be same as event.target and the event was dispatched only after the actual DOM scroll values were changed. But the layout could get invalidated due to some other reason and reading scrollLeft would cause reflow. I am only concerned with number of callbacks we get here per sec. What do you think?